catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/Relationship.java

```diff
@@ -1,111 +0,0 @@
-/*
- *  Copyright 2021 Collate
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *  http://www.apache.org/licenses/LICENSE-2.0
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-
-package org.openmetadata.catalog.jdbi3;
-
-/**
- * This enum captures all the relationships between Catalog entities Note that the relationship from is a Strong entity
- * and to is Weak entity when possible.
- *
- * <p>Note: Rules for changing enums since the ordinal position is stored in the database. - Don't remove an enum, since
- * the database might have stored the enum ordinal number - When adding a new enum, add it as the last enum to preserve
- * the ordinal positions of the existing enums
- */
-public enum Relationship {
-
-  /**
-   * CONTAINS relationship is a stronger relationship than HAS. The entity that contains other entities can't be deleted
-   * until all the entities that it contains are also deleted. Some examples of these relationships:
-   *
-   * <ul>
-   *   <li>Database --- contains --> Table
-   *   <li>DatabaseService --- contains --> Database
-   *   <li>MessagingService --- contains --> Topic
-   *   <li>PipelineService --- contains --> Pipeline
-   *   <li>DashboardService --- contains --> Charts
-   *   <li>DashboardService --- contains --> Dashboard
-   *   <li>Role --- contains --> Policy
-   * </ul>
-   */
-  CONTAINS("contains"), // 0
```

Comment:
@harshach 

I kept the enum case and values exactly the same as what we had before.

```java
@Generated("jsonschema2pojo")
public enum Relationship {

    CONTAINS("contains"),
    CREATED("createdBy"),
    REPLIED_TO("repliedTo"),
    IS_ABOUT("isAbout"),
    ADDRESSED_TO("addressedTo"),
    MENTIONED_IN("mentionedIn"),
    TESTED_BY("testedBy"),
    USES("uses"),
    OWNS("owns"),
    PARENT_OF("parentOf"),
    HAS("has"),
    FOLLOWS("follows"),
    JOINED_WITH("joinedWith"),
    UPSTREAM("upstream"),
    APPLIED_TO("appliedTo");
    private final String value;
    private final static Map<String, Relationship> CONSTANTS = new HashMap<String, Relationship>();

```

This was done so that we can avoid breaking anything